sdks/python/apache_beam/runners/worker/operations.py

```diff
@@ -812,7 +812,7 @@ def __init__(self,
 
     # See fn_data in dataflow_runner.py
     # TODO: Store all the items from spec?
-    self.fn, _, _, _, _ = (pickler.loads(self.spec.serialized_fn))
+    self.fn, _, _, _, _ = pickler.loads(self.spec.serialized_fn)
```

Comment:
This looks like a Cython bug.

Reply:
Ah that was a linting-related change. Cython didn't flag that line, the closure was the only code that cython was explicitly unhappy with